README.md

```diff
@@ -115,11 +115,7 @@ Specifically, if adding more than one new domain, please cite in your PR where o
 
 Use:
 
```

Comment:
I think these lines need removed as well.

Reply:
Done! Sorry for the redundancy.